app/workers/certificates.php

```diff
@@ -108,90 +171,140 @@ public function run(): void
             throw new Exception('You must set a valid security email address (_APP_SYSTEM_SECURITY_EMAIL_ADDRESS) to issue an SSL certificate');
         }
 
-        $stdout = '';
-        $stderr = '';
-
-        $exit = Console::execute("certbot certonly --webroot --noninteractive --agree-tos{$staging}"
-            . " --email " . $email
-            . " -w " . APP_STORAGE_CERTIFICATES
-            . " -d {$domain->get()}", '', $stdout, $stderr);
-
-        if ($exit !== 0) {
-            throw new Exception('Failed to issue a certificate with message: ' . $stderr);
-        }
-
-        $path = APP_STORAGE_CERTIFICATES . '/' . $domain->get();
-
-        if (!\is_readable($path)) {
-            if (!\mkdir($path, 0755, true)) {
-                throw new Exception('Failed to create path...');
+        try {
```

Comment:
Big try-catch block are not healthy. We have an error handler for workers, can we use it?